### DISCUSSION TITLE
v1.8.0 - Maintenance & Plugin Update

### DISCUSSION THREAD

**Discussion by iwalton3** [jellyfin/jellyfin-desktop#370] [General]
(This release has been replaced by v1.8.1 due to #371.)

**Comment by digitalbarrito** [comment]
Thank you an incredible amount for even the experimental support for Intro Skipper and Jellyscrub. I fully understand it's not a commitment for the future, but I do fully hope that it continues as supported plugins in the future, they are wonderful QOL additions, especially because I run on a Pi through kubernetes and the unofficial images that include the tweaks to run these pllugins don't have Pi images.

**Comment by Muffinut** [comment]
I'm lucky to see this update on the day I started using Jellyfin! I really hope web client 10.9 is out soon to customize subs.
